oasislmf/utils/data.py

```diff
@@ -1042,3 +1048,41 @@ def set_dataframe_column_dtypes(df, dtypes):
     df = df.astype(_dtypes)
 
     return df
+
+
+def fill_na_with_categoricals(df, fill_value):
```

Comment:
As this function is "public"/ imported in summaries, can you add it the the list in __all__ at the top of the file